widget/app.js

```diff
@@ -439,6 +443,20 @@
                 $rootScope.$emit('deviceLocked', {});
                 //callPlayer('ytPlayer', 'pauseVideo');
             });
+
+            $rootScope.hideNavbar = function () {
```

Comment:
Move logic to routeChangeSuccess

if (path === 'nowPlaying') showNavbar()
else hideNavbar()